.babelrc

```diff
@@ -1,4 +1,8 @@
 {
-  "comments": false,
-  "loose": "all"
+  "presets": [
+    ["env", {
```

Comment:
does loose fall to es2015 automatically? I thought it had to be nested under `{ es2015: { loose: true } }`